pages/print_labels/ad_hoc_plate.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <b-container>
+    <b-row>
+      <b-col>
+        <h1>Print Control plate labels</h1>
+        <p class="lead"></p>
+
+        <!-- TODO: better in a component of its own? -->
+        <p>
+          <b-alert :show="isError" dismissible variant="danger">
+            {{ alertMessage }}
+          </b-alert>
+          <b-alert :show="isSuccess" dismissible variant="success">
+            {{ alertMessage }}
+          </b-alert>
+          <b-alert :show="isBusy" dismissible variant="warning">
+            {{ alertMessage }}
+          </b-alert>
+        </p>
+        <p>
+          <label for="selectPrinter">
```

Comment:
A Printer Select component would be a good one to abstract, think it is being used in a few places